ci/lib/stage-test-examples.jenkinsfile

```diff
@@ -195,7 +199,7 @@ stage('test-examples') {
             } catch (Exception e) { }
         }
 
-        if (["ubuntu20.04", "ubuntu22.04", "debian11"].contains(env.base_os)) {
+        if (["ubuntu20.04", "ubuntu22.04", "debian11"].contains(env.base_os) || (env.base_os != "ubuntu24.04")) {
```

Comment:
We don't here special condition for Ubuntu 24.04 because the if logic will return false if distro is ubuntu 24.04

Reply:
Modified